{"name": "test_Visit", "status": "broken", "statusDetails": {"message": "NameError: name 'driver' is not defined", "trace": "@allure.story('進入SSO登入頁面')\n    @pytest.mark.run(order=3)\n    def test_Visit():\n    \n>           driver.get(\"https://qa.telli.cc/consumer/oauth2/login?ReturnUrl=%2Fconsumer%2Foauth2%2Fconnect%2Fauthorize%3Fclient_id%3Dauthorization-code-client%26redirect_uri%3Dhttps%253A%252F%252Fqa.telli.cc%252Fconsumer%252Fmember%252Fcallback%26response_type%3Dcode%26scope%3D%2520%26state%3D3e390076cc644b3788ed41ed39f2d69d%26code_challenge%3DIlR89jtBkm3IK3MQ4GEat-rfLQAIfsyZ8pRaw_nYbkg%26code_challenge_method%3DS256%26response_mode%3Dquery%26companyId%3D97d288ac-5f17-11ed-afa6-00ffaf2156c9\")\nE           NameError: name 'driver' is not defined\n\nSSO_Line_test.py:55: NameError"}, "start": 1669948640214, "stop": 1669948640214, "uuid": "055a99e5-30d9-46eb-b260-a9b0670d2bae", "historyId": "6bdc735a21b42e69c97ae3a5c15fe9ea", "testCaseId": "f61b388a2b2e4c51375b58ffa51fecb4", "fullName": "SSO_Line_test#test_Visit", "labels": [{"name": "story", "value": "進入SSO登入頁面"}, {"name": "tag", "value": "run(order=3)"}, {"name": "suite", "value": "SSO_Line_test"}, {"name": "host", "value": "C010700042"}, {"name": "thread", "value": "16756-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "SSO_Line_test"}]}